src/backend/core/api/viewsets.py

```diff
@@ -883,7 +882,7 @@ def perform_create(self, serializer):
         """Save invitation to a document then send an email to the invited user."""
         invitation = serializer.save()
 
-        language = self.request.headers.get("Content-Language", "en-us")
+        language = settings.LANGUAGE_CODE
```

Comment:
There is big chances that the new user will have the same language as the sender, I will put the sender language in this case.
```suggestion
        language = self.request.user.language
```